Dockerfile.dev

```diff
@@ -1,5 +1,8 @@
 FROM erlang:24-alpine
 
+ENV PUID=1000
+ENV PGID=1000
```

Comment:
Why default to 1000?  Shouldn't this be derived from the current user?

The `stat` construct is to fix access permission errors on the directories and files accessed by the container, when accessing and updating files owned by the user running the container. For this we want to use the same UID and GID in the container as the current user.

Reply:
The problem is as following:
The directory opt/zotonic is not present on a clean install, so it is created by the default user the base image is using; the root user. 
The docker/docker-entrypoint.sh script then performs a ```stat``` on the opt/zotonic directory, and get 0:0, the root user's UID and GID.
It then tries to create the user zotonic with UID and GID 0, which fails of course.

To reproduce: delete the zotonic volume, and run ```./start-docker.sh```.

Fixing it in the docker-compose file works too, but then you will still not be able to build the image using something like ```docker build -f Dockerfile.dev -t zotonic/zotonic-dev:latest .```. I do not know if that is ok.

Reply:
I am wondering why the directory `opt/zotonic` is not present on clean install.
Isn't it mounted before the install script?